### DISCUSSION TITLE
ostrich password

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#84] [Q&A] [answered]
Did the password for ostrich change? I'm trying to transfer the data I have on ostrich to gannet so it can be available publicly and I've tried the one that was sent out in the week 3 announcement that previously worked fine (just last week even) but now its not. I'm still using the same user from the announcement as well.

**Comment by kubu4** [comment]
It has not changed. Will try to look into this in a bit and report back.

**Comment by kubu4** [comment]
Also, please post a screenshot of the command you're running and the error message you're receiving.

**Reply by meganewing** [reply]
<img width="563" alt="Screen Shot 2021-02-18 at 3 17 21 PM" src="https://user-images.githubusercontent.com/66333757/108434430-7c894400-71fc-11eb-9d50-22d4ae2a3847.png">

I'm not getting an error message but since it keeps asking for it I'm assuming its wrong. I've tried multiple times and I'm 99% sure theres no typos.

**Comment by kubu4** [comment]
@sr320 Looks like the fish546 user account no longer exists on Ostrich. Any ideas? I guess we could just create a new one, but I suspect people's data are probably gone...

![image](https://user-images.githubusercontent.com/4514104/108434874-37194680-71fd-11eb-9a21-b66b564687dc.png)

**Comment by kubu4** [comment]
@sr320 I can't seem to remote desktop in to Ostrich, but here is a guide to restore a deleted user account (can't find any instructions for using Terminal to do this):

https://support.apple.com/guide/mac-help/restore-a-deleted-user-mchlp2968/mac

Would you have time to do the first set of instructions (restore from disk image)?

**Comment by sr320** [comment, marked as answer]
All files will be at
https://gannet.fish.washington.edu/fish546_2021/Files_Giles_Uploaded_For_Chris/
in about 30 minutes
